sorc/ncep_post.fd/UPP_PHYSICS.f

```diff
@@ -1399,6 +1399,10 @@ SUBROUTINE CALCAPE2(ITYPE,DPBND,P1D,T1D,Q1D,L1D,    &
         ENDDO
       ENDDO
 !
+!Ensure later calculations do not access LM+1
```

Comment:
@SamuelTrahanNOAA Could you add a change log with your name at the beginning session of subroutine CALCAPE2?

Reply:
Added.